packages/cli/src/tests/reloader.test.ts

```diff
@@ -0,0 +1,81 @@
+import { describe, expect, test } from 'vitest'
+import { getCurrentFileDirectory } from '@prosopo/util'
+import { promisify } from 'util'
+import { spawn } from 'child_process'
+import fs from 'fs'
+import path from 'path'
+
+describe('reloading api', () => {
+    test.only('api reloads after changing .env file', async () => {
```

Comment:
what's the reason you're using `only` here?

Reply:
it's a mistake.